src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const stringSplitted = sourceString.split(';');
+
+  const filtered = stringSplitted.filter((item) => item.trim().length);
+
+  const dividedWords = filtered.map((words) => words.split(':'));
+
+  dividedWords.forEach(([key, value]) => {
```

Comment:
[CODE KNOWLEDGE] - Good job using destructuring in the forEach loop to make your code cleaner and more readable.